lib/gamelogic.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/LineLength
+# rubocop:disable Metrics/CyclomaticComplexity
+class Game
+  attr_accessor :players
+  def initialize
+    @players = []
+  end
+
+  def toset_player(player)
+    @players << player
+  end
+end
+
+class Board
+  attr_accessor :counter
+  attr_reader :result, :contador, :winner
+
+  def initialize
+    @board = { a1: '', a2: '', a3: '', b1: '', b2: '', b3: '', c1: '', c2: '', c3: '' }
+    @turn = 'X'
+    @result = ''
+    @contador = 0
+    @winner = ''
+    @counter = 1
+  end
+
+  def mark_choice(choice, _counter)
+    @contador += 1
+    @turn = @counter == 1 ? 'X' : 'O'
+    @board[choice.to_sym] = 'X' if @turn == 'X'
+    @board[choice.to_sym] = 'O' if @turn == 'O'
+    check_victory
+  end
```

Comment:
Instead of using string `'X'` and `'O'`, please use symbol, I mean `:X` and `:O`